code/game/machinery/cryopod.dm

```diff
@@ -0,0 +1,464 @@
+					to_chat(O.owner.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
+				O.target = null
+				spawn(1) //This should ideally fire after the occupant is deleted.
+					if(!O) return
+					O.find_target()
+					if(!(O.target))
+						O.owner.objectives -= O
+						qdel(O)
+	if(mob_occupant.mind && mob_occupant.mind.assigned_role)
+		//Handle job slot/tater cleanup.
+		var/job = mob_occupant.mind.assigned_role
+		SSjob.FreeRole(job)
+		if(mob_occupant.mind.objectives.len)
+			mob_occupant.mind.objectives.Cut()
+			mob_occupant.mind.special_role = null
+
+	// Delete them from datacore.
+
+	var/announce_rank = null
+	for(var/datum/data/record/R in GLOB.data_core.medical)
+		if((R.fields["name"] == mob_occupant.real_name))
+			qdel(R)
+	for(var/datum/data/record/T in GLOB.data_core.security)
+		if((T.fields["name"] == mob_occupant.real_name))
+			qdel(T)
+	for(var/datum/data/record/G in GLOB.data_core.general)
+		if((G.fields["name"] == mob_occupant.real_name))
+			announce_rank = G.fields["rank"]
+			qdel(G)
+
+	for(var/obj/machinery/computer/cloning/cloner in world)
+		for(var/datum/data/record/R in cloner.records)
+			if(R.fields["name"] == mob_occupant.real_name)
+				cloner.records.Remove(R)
+
+	//Make an announcement and log the person entering storage.
+	control_computer.frozen_crew += "[mob_occupant.real_name]"
+
+	if(GLOB.announcement_systems.len)
+		var/obj/machinery/announcement_system/announcer = pick(GLOB.announcement_systems)
+		announcer.announce("CRYOSTORAGE", mob_occupant.real_name, announce_rank, list())
+		visible_message("<span class='notice'>\The [src] hums and hisses as it moves [mob_occupant.real_name] into storage.</span>")
+
+	// Ghost and delete the mob.
+	if(!mob_occupant.get_ghost(1))
+		if(world.time < 30 * 600)//before the 30 minute mark
+			mob_occupant.ghostize(0) // Players despawned too early may not re-enter the game
+		else
+			mob_occupant.ghostize(1)
+	QDEL_NULL(occupant)
+	open_machine()
+	name = initial(name)
+
+/obj/machinery/cryopod/MouseDrop_T(mob/living/target, mob/user)
+	if(!istype(target) || user.incapacitated() || !target.Adjacent(user) || !Adjacent(user) || !ismob(target) || (!ishuman(user) && !iscyborg(user)) || !istype(user.loc, /turf) || target.buckled)
+		return
+
+	if(occupant)
+		to_chat(user, "<span class='boldnotice'>The cryo pod is already occupied!</span>")
+		return
+
+	if(target.stat == DEAD)
+		to_chat(user, "<span class='notice'>Dead people can not be put into cryo.</span>")
+		return
+
+	if(target.client && user != target)
+		to_chat(user, "<span class='notice'>You can't put conscious people into [src]!</span>")
+	else if(target.client)
+		if(alert(target,"Would you like to enter cryosleep?",,"Yes","No") == "No")
+			return
+
+	var/generic_plsnoleave_message = " Please adminhelp before leaving the round, unless there are no administrators online."
```

Comment:
`unless` -> `even`

We'll be outputting ahelps to discord... or the mins could be invisimin